### PR TITLE
Added alternative way to store keystore passwords for macOS users

### PR DESCRIPTION
**Pull request by sonnyvesali**
_Description of what this PR is changing or adding, and why:_
Added a site note including the link to a blog post that shows macOS users how they can store keystore passwords in the Keychain app instead of plain text.
_Issues fixed by this PR (if any):_

## Presubmit checklist

- [X] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [X] This PR follows the [Google Developer Documentation Style Guidelines](  c) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [X] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by parlough** [comment]
Thanks for opening a pull request! However, before we can take a look and review it, you'll need to sign the CLA as outlined at https://github.com/flutter/website/pull/8818/checks?check_run_id=13941071948. Let us know if you have any issues, thanks!

**Comment by sonnyvesali** [comment]
> Thanks for opening a pull request! However, before we can take a look and review it, you'll need to sign the CLA as outlined at https://github.com/flutter/website/pull/8818/checks?check_run_id=13941071948. Let us know if you have any issues, thanks!

@parlough I tried signing it but it won't let me put a non-google email and my github is under a non-google account, how can I get around this obstacle?

**Comment by sonnyvesali** [comment]
> Thanks for opening a pull request! However, before we can take a look and review it, you'll need to sign the CLA as outlined at https://github.com/flutter/website/pull/8818/checks?check_run_id=13941071948. Let us know if you have any issues, thanks!

@parlough is it not possible to sign the CLA with a non-google github account? I used my gmail and then put my GH username, I'm not sure it fixed it, lmk what else I need / can do.

File: src/deployment/android.md [hunk 1]
```diff
@@ -192,6 +192,13 @@ by editing the `[project]/android/app/build.gradle` file.
 
 </li>
 
+{{site.alert.note}}
+  Note: If you'd prefer not to store your passwords in plain text
+  you can store your credentials [in the Keychain Access App](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/).
+  In which case you can skip the first 2 lines of `key.properties`
+  file and follow the steps in the link above accordingly.
```
Comment (sfshaza2) [inline src/deployment/android.md:199]:
```suggestion
  If you'd prefer not to store your passwords in plain text,
  you can store your credentials in the [Keychain Access App][].
  In this case, skip the first 2 lines of `key.properties`
  file and follow the steps in the Pilloxa link.
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Please make these minor changes and I can lgtm.

**Comment by sfshaza2** [comment]
@sonnyvesali, the CLA test is looking for a signature from:  sonny.vesali@proton.me. It seems that your account isn't set up properly.

**Comment by sfshaza2** [comment]
Did we lose you, @sonnyvesali? I'm sorry about the confusion with the CLA.

**Comment by parlough** [comment]
Thanks again for the pull request!

As mentioned earlier, you'll need to sign the CLA with the proton email for us to take a look. Sorry about the confusion, but let us know if you need any help doing so :) 

I'll close this for now, but feel free to reopen once you sign the CLA or open a new PR. Thanks!

**Pull request closed without merging**